pep-0501.txt

```diff
@@ -413,65 +547,83 @@ printf-style formatting. The runtime parsing and interpolation overhead for
 
 This kind of approach would be possible as having access to the actual *text*
 of the field expression would allow the logging renderer to distinguish
 between inline tuples that appear in the field expression itself, and tuples
 that happen to be passed in as data values in a normal field.
 
 
+Comparison to PEP 675
+=====================
+This PEP is similar in its goals as :pep:`675`.
+While both are attempting to provide a way to have safer code, they are doing so in different ways.
+:pep:`675` provides a way to find potential security issues as a static analysis.
+It does so by providing a way for the type checker to flag sections of code that are using
+dynamic strings incorrectly. This requires a user to actually run a static analysis type checker such as mypy.
+
+If :pep:`675` tells you that you are violating a type check, it is up to the programmer to know how to handle the dynamic-ness of the string.
+This PEP provides a safe alternative to f-strings at runtime.
+If a user recieves a type-error, changing an existing f-string into a t-string could be an easy way to solve the problem.
+
+t-strings create safer code by correctly escaping the dynamic sections of strings, while maintaining the static portions.
+
+This PEP also allows a way for a library/codebase to be safe, but it does so at runtime rather than
+only during static analysis. For example, if a library wanted to ensure "only safe strings", it
+could check that the type of object passed in at runtime is a template literal.::
+
+    def my_safe_function(string_like_object):
+        if not isinstance(string_like_object, types.TemplateLiteral):
+            raise TypeError("Argument 'string_like_object' must be a t-string")
+
+The two PEPs could also be used together by typing your function as accepting either a string literal or a template literal.
+This way, your function can provide the same api for both static strings, and dynamic strings.
+
+    def my_safe_function(string_like_object: LiteralString | TemplateLiteral):
+
 Discussion
 ==========
 
 Refer to :pep:`498` for additional discussion, as several of the points there
 also apply to this PEP.
 
-Deferring support for binary interpolation
-------------------------------------------
-
-Supporting binary interpolation with this syntax would be relatively
-straightforward (the elements in the parsed fields tuple would just be
-byte strings rather than text strings, and the default renderer would be
-markedly less useful), but poses a significant likelihood of producing
-confusing type errors when a text renderer was presented with
-binary input.
+Support for binary interpolation
+--------------------------------
 
-Since the proposed syntax is useful without binary interpolation support, and
-such support can be readily added later, further consideration of binary
-interpolation is considered out of scope for the current PEP.
+As :pep:`498` does not handle byte strings, neither will this one.
```

Comment:
```suggestion
As f-strings don't handle byte strings, neither will t-strings.
```

Use the far more widely known implemented feature name rather than the historical PEP number